README.md

```diff
@@ -49,20 +49,30 @@ And to declare custom paste command use:
 let g:system_copy#paste_command='xclip -sel clipboard -o'
 ```
 
+To make the original yank vim command to works with this plugin, one can add the following to
+it's `.vimrc`:
+
+```
+" Mapping Y to use copy
+nmap Y <Plug>SystemCopy
+" Mapping P to use paste
+nmap P <Plug>SystemPaste
+```
```

Comment:
I like the idea of documenting custom key bindings, but to revisit what I said in my first comment, the goal is to avoid replacing `y` and `p`. Can you rewrite this in terms of a leader mapping? Something like:

``` md
If you'd rather not use the default mappings, you can configure
custom mappings instead:

``` vim
# Use <leader> maps instead of the default key maps
nmap <leader>cp <Plug>SystemCopy
nmpa <leader>cv <Plug>SystemCopy
```
```

Reply:
Good idea, I think I mess some concepts myself, you're completely right, sorry for that. Fixed this on the in e9fb675